media/js/firefox/australis/fx36/firstrun.js

```diff
@@ -0,0 +1,64 @@
+;(function($, Mozilla) {
+    'use strict';
+
+    var helloAnimationStage = $('#hello-animation-stage');
+
+    function trackDefaultSearchEngine() {
```

Comment:
@ckprice It would also be good to know if this is still needed in firstrun. It was added in FF 34 for the search experiments, but may no longer be needed now. Seeing as it's setting FHR data, we should only really do it if absolutely necessary.